OpenSim/Common/Object.cpp

```diff
@@ -1647,19 +1647,17 @@ void Object::updateFromXMLDocument()
     IO::chDir(saveWorkingDirectory);
 }
 
-std::string Object::dump(bool dumpName) {
+std::string Object::dump() const {
     SimTK::String outString;
```

Comment:
While this change could be reasonable by itself, folding it into this PR is unnecessary and will be confusing. Also breaks backward compatibility.